src/services/codefixes/helpers.ts

```diff
@@ -899,8 +899,14 @@ export function tryGetAutoImportableReferenceFromTypeNode(importTypeNode: TypeNo
         if (isLiteralImportTypeNode(node) && node.qualifier) {
             // Symbol for the left-most thing after the dot
             const firstIdentifier = getFirstIdentifier(node.qualifier);
-            const name = getNameForExportedSymbol(firstIdentifier.symbol, scriptTarget);
-            const qualifier = name !== firstIdentifier.text
+            if (!firstIdentifier.symbol) {

```

Comment:
Value-space identifiers get their `.symbol` in the binder but type-level identifiers in import type nodes don't.

Alternatively, this could be fixed in `visitExistingNodeTreeSymbolsWorker` but it's more tricky there.